server/services/TransferService.js

```diff
@@ -115,6 +202,31 @@ class TransferService {
 
       await this._session.commitTransaction();
 
+      if (transfer) {
+        const originator_wallet_id = await this._walletService.getByName(
+          transfer.originating_wallet,
+        );
+        const destination_wallet_id = await this._walletService.getByName(
+          transfer.destination_wallet,
+        );
+
+        // transfer completed
+        // the log should show up on both sender and receiver
+        await this._event.logEvent({
```

Comment:
These should also be part of the knex transaction

Reply:
Sorry overlooked all the knex transactions calls! Will include all the logEvent functions within the knex transactions